src/python/tensorflow_cloud/core/containerize.py

```diff
@@ -157,32 +157,19 @@ def _create_docker_file(self):
                 if float(v[0] + "." + v[1]) <= 2.1:
                     self.docker_base_image += "-py3"
 
-        if not self._base_image_exist():
+        if "dev" in self.docker_base_image:
```

Comment:
How safe is this to rely on this sub-string? Would be better to leave a comment describing what image URI is this check is expecting.

Reply:
All TF nightlies and only nightlies have this substring. Added a comment.